README.md

```diff
@@ -1,2 +1,23 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Microverse-WorkFlow1
-Set up a "Hello Microverse" project following a specific work flow
+Set up a "Hello Microverse" project following a specific Microverse work flow
+
```

Comment:
- [ ] Kindly make sure your project has a descriptive `title` and `detailed description` saying what the project is about. :smile: 
Hint: You use the ones provided in the project requirements